src/services/AudiusBackend.js

```diff
@@ -452,6 +455,7 @@ class AudiusBackend {
           IDENTITY_SERVICE
         ),
         creatorNodeConfig: AudiusLibs.configCreatorNode(USER_NODE, true),
+        comStockConfig: AudiusLibs.configComStock(COM_STOCK_URL),
```

Comment:
not in this PR, but this should be an optional config in libs, just double checking